software_supply_chain_attacks.csv

```diff
@@ -69,19 +72,26 @@ unknown,11/5/17,11/5/17,4,Google Play Store,Fake versions of Whatsapp were avail
 1/1/18,unknown,7/26/18,1,PDF editor application,"Attackers compromised a font package installed by a PDF editor application and used it to deploy a cryptocurrency miner on users' computers. Since the PDF editor was installed under SYSTEM privileges, the malicious coinminer code hidden inside the font package would receive full access to the victims' system.",https://www.microsoft.com/security/blog/2018/07/26/attack-inception-compromised-supply-chain-within-a-supply-chain-poses-new-risks/,,0,0,0,0,0,0,1,0,0,Software Registry Attacks,Dependency Compromise
 unknown,unknown,7/30/18,145,Google Play Store,there are 145 Google Play apps infected by malicious Microsoft Windows executable files on the Google Play Store. Palo Alto Networks reported the findings to Google Security Team and all infected apps have been removed from Google Play.,https://unit42.paloaltonetworks.com/unit42-hidden-devil-development-life-cycle-google-play-apps-infected-windows-executable-files/,,0,0,0,0,0,0,0,1,0,Software Registry Attacks,Malicious Package
 unknown,7/31/18,8/21/18,1,South Korean remote support solutions provider,The threat actors compromised the update server of a remote support solutions provider to deliver a remote access tool called 9002 RAT to their targets of interest through the update process. They carried this out by first stealing the company’s certificate then using it to sign the malware. They also configured the update server to only deliver malicious files if the client is located in the range of IP addresses of their target organizations.,https://blog.trendmicro.com/trendlabs-security-intelligence/supply-chain-attack-operation-red-signature-targets-south-korean-organizations/,,0,0,0,1,1,0,0,0,0,"Build, Source, and Publishing Infrastructure Attack",Publishing Infrastructure: Delivery System Compromise
+,8/16/18,,23,Mozilla Add-Ons,"Mozilla removes 23 malicious extensions caught spying (including ""Web Security"")","Cimpanu, C. (2018). Mozilla Removes 23 Firefox Add-Ons That Snooped on Users. BleepingComputer. Retrieved from https://www.bleepingcomputer.com/news/security/mozilla-removes-23-firefox-add-ons-that-snooped-on-users",,0,0,0,0,0,1,0,1,0,Software Registry Attacks,Malicious Package
+,8/24/18,,2,USB Stick,Schneider Electric battery monitor USB infected with malware (2 SKUs),"Security Notification � USB Removable Media Provided With Conext Combox and Conext Battery Monitor | Schneider Electric. (2020, January 03). Retrieved from https://www.se.com/ww/en/download/document/SESN-2018-236-01",contaminated with malware during manufacturing at one of schneider's suppliers.,0,0,1,0,0,0,0,0,0,"Build, Source, and Publishing Infrastructure Attack",Firmware Implant
```

Comment:
@jaxley, and the unicode character here too, please.

Reply:
Regardless, we'll make a best guess of the symbols there. Thank you for the help! Please let us know if you're ever interested in further collaboration.